sdk/cosmos/azure-cosmos/CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 ### 4.3.1 (Unreleased)
 
 #### Features Added
+- GA release of integrated cache functionality [See PR #25391](https://github.com/Azure/azure-sdk-for-python/pull/25391).
```

Comment:
https://azure.github.io/azure-sdk/policies_releases.html#change-logs
The guidelines don't say _don't_ do this, per se, but I would recommend against it. Generally, customers aren't going to look at Github PRs. In the guidelines linked, we only put links to GH issues for fixed bugs.

Reply:
@simorenoh since this seems diametrically opposed to your PR feedback

Reply:
yeah this is a practice that we recently started implementing in the other SDKs to make it easier for us to pinpoint sources of regressions by having the link to the PR associated with those changes available. It's more a thing for us than it is for the customers